imagedl.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+
+import requests
+import re
+import argparse
+import os
+import rosters
+
+TITLE_ART = """
+         ____                    ___  __ 
+        /  _/_ _  ___ ____ ____ / _ \/ / 
+       _/ //  ' \/ _ `/ _ `/ -_) // / /__
+      /___/_/_/_/\_,_/\_, /\__/____/____/
+                     /___/               
+                     """
+
+YELLOW = "\033[1;33m"
+LIGHT_BLUE = "\033[1;34m"
+END = "\033[0m"
+
+def multiline_input():
+    lines = []
+    while True:
+        inpt = input()
+        if len(inpt) > 0:
+            lines.append(inpt)
+        else:
+            return lines
+
+
+def parse_curl(curl):
+  url = re.search("'([^']*)'", curl[0]).group(1)
+
+  headers = {}
+  for line in curl[1:]:
+      if line.strip().startswith('-H'):
+          header_name, header_value = re.search("'([^']*)'", line).group(1).split(': ')
+          headers[header_name] = header_value
+
+  return url,headers
+
+def do_dl(url_prefix, headers, unis, output_dir):
+    counter = 0
+    uni_count = len(unis)
+    for uni in unis:
+        url = url_prefix+uni+'.jpg'
+        
+        counter += 1
+        percentage = f"{round(counter/uni_count*100, 1)}%"
+        print("Downloading", url, f"({counter}/{uni_count} -- {percentage})")
+        response = requests.get(url, headers=headers)
+
+        with open(os.path.join(output_dir, uni+'.jpg'), 'wb') as f:
+            f.write(response.content)
+        
+last_step_number = 0
+def step(instruction):
+    global last_step_number
+
+    print(f'[{last_step_number}]', instruction)
+    last_step_number += 1
+    input()
```

Comment:
`input()` takes a prompt, can just put the print contents in that using an f-string